changelog.d/1105.change.md

```diff
@@ -0,0 +1 @@
+Added instructions for pull request workflow to CHANGELOG.md
```

Comment:
Not sure if the change note is useful for this patch but if it is, I'd probably mark the filename as inline code.
```suggestion
Added instructions for pull request workflow to `CHANGELOG.md`.
```

Reply:
> Not sure if the change note is useful for this patch 

How about adding a [custom fragment](https://towncrier.readthedocs.io/en/stable/configuration.html#custom-fragment-types) like `docs` for towncrier?

Reply:
Changelog represents changes between versions, it should show how those changes effect the end-users.
This patch does not affect the end-users at all, only contributors. Honestly, the docs fragment type is rather debatable by itself — I think it's rare for it to be useful.

Reply:
Ok, thanks for your feedback!